DEVELOPMENT.md

```diff
@@ -65,15 +117,24 @@ ACCOUNT_FACTORY_ACCOUNT_ID=... \
 
 #### Login to the test environment
 
+This uses `firefox` and logs you in to the test account. You get the `AWS_ACCOUNT_ID` as follows
+1. In the GitHub PR click in the *Details* link of the `AWS CodeBuild BuildBatch eu-west-1` check
+2. Click on the `uuid` of the `Build run`, where you find the `arn` of the `OVMCrossAccountRole`, which contains the AccountID, where the fresh superwerker installation runs.
+3. If the build logs are not present any more due to the retention time, then take the `Build number` and look for the details in the [BuildAndTestProject](https://eu-west-1.console.aws.amazon.com/codesuite/codebuild/projects?region=eu-west-1) codebuild project
+ 
 ```bash
 SUPERWERKER_REGION=uk-east-1 \
```

Comment:
typo us-east-1

Reply:
is intentional so people have to adapt it after copy-pasting